src/Linter/LintingException.php

```diff
@@ -16,6 +16,8 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @final
```

Comment:
i got confused - is this indeed changing the coverage?
or it's only a side change?

Reply:
side change, I can extract it in another PR if needed

Reply:
ok, no worry - i thought that maybe somehow it was impacting the coverage.

other topic then, just opening the pandora box... isn't marking class final a bc break? maybe annotation is good, but then we can add TODO for next MAJOR to move annotation to proper `final class` declaration

Reply:
I think making the class final is a BC break, but adding an annotation I'm not sure, I'll take it out than it can be discussed some more

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6021